src/main/java/apoc/export/graphml/XmlGraphMLWriter.java

```diff
@@ -65,11 +69,13 @@ private void writeKeyTypes(XMLStreamWriter writer, Map<String, Class> keyTypes,
             writer.writeAttribute("id", entry.getKey());
             writer.writeAttribute("for", forType);
             writer.writeAttribute("attr.name", entry.getKey());
-            if (typeClass.isArray()) {
-                writer.writeAttribute("attr.type", "string");
-                writer.writeAttribute("attr.list",type);
-            } else {
-                writer.writeAttribute("attr.type", type);
+            if (useTypes) {
+                if (typeClass.isArray()) {
+                    writer.writeAttribute("attr.type", "string");
```

Comment:
puoi fare una prova e metterlo così
```suggestion
                    writer.writeAttribute("attr.type", type);
```
Vedi cosa succede